docs/community/glossary.md

```diff
@@ -0,0 +1,15 @@
+| napari term        | Description                                                                                                                                                                                                                                                                                                                                 |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| channel            | A channel is a grayscale image. In fluorescence microscopy,  each channel corresponds to a range of wavelengths. An RGB image from digital camera consists of 3 channels - red, green, blue.                                                                                                                                                |
+| color map          | An array of colors used to map pixel data to the actual color values.                                                                                                                                                                                                                                                                       |
+| dock widget        | A widget that can be attached to the main napari viewer at a specified position.                                                                                                                                                                                                                                                            |
```

Comment:
* And what's a widget? You'll need to explain that here too.
* Maybe it's worth emphasizing that attached is a synonym for docked in this context, by putting the word next to in in brackets? *"...that can be attached (docked) to the ..."*

Reply:
```suggestion
| dock widget        | A panel of the GUI interface that can be attached to the main napari viewer at specific positions e.g to the right of the canvas or underneath it.                                                                                                                                                                                                                                                            |
```